code/modules/antagonists/space_dragon/space_carp.dm

```diff
@@ -0,0 +1,32 @@
+/datum/antagonist/space_carp
+	name = "\improper Space Carp"
+	antagpanel_category = ANTAG_GROUP_LEVIATHANS
+	show_in_roundend = FALSE
+	show_in_antagpanel = FALSE
+	show_name_in_check_antagonists = TRUE
+	show_to_ghosts = TRUE
+	/// The rift to protect
+	var/obj/structure/carp_rift/rift
```

Comment:
where is this used?

Reply:
assigned at creation of a mob controlled carp, then assigned to the protect obj